erica/application/Shared/base_service.py

```diff
@@ -0,0 +1,13 @@
+from uuid import UUID
+from erica.application.erica_request.erica_request_service import EricaRequestService
+
+
+class BaseService:
+    erica_request_service: EricaRequestService
+
```

Comment:
Could we add a type here, that is None for the base class and would be RequestType.freischaltcode_activate etc. for the other classes. But we can also add that in a separate PR

Reply:
What do we need type for in the subclasses?

Reply:
This was just an idea how we couldset the correct request types for all the different request. However, we could also put this as a parameter into the function. Nevertheless, this probably does not make sense to do here but directly when we implement raising an error in case the type was not found.

Reply:
oh true you we thinking a bit further regarding your PR with the error handling. Let me take over in your PR after we merge this one